tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -89,6 +89,7 @@ Define {
 	BT_PB_HOST_PIPELINE_SINK "copier.SSP.10.1"
 	BT_PB_DAI_PIPELINE_SRC "copier.host.9.1"
 	BT_PB_PIPELINE_STREAM_NAME "copier.SSP.10.1"
+	PASSTHROUGH	"false"
```

Comment:
@kv2019i while adding a new passthrough topology file seems like an easier solution at the outset, it is harder to maintain. There are quite a few details that are difficult to track in terms of the DAI configuration and maintaining this across multiple files will just not be feasible. Taking the existing nocodec tplg and bypassing the processing pipelines is the simplest solution even though it involves a bit of work right now.

Reply:
I am not so sure @ranj063. 

The drawback is that we end-up pushing conditional parts all the way down in the hope of maintaining a single top-level file. That's what prompted me to just get rid of everything and use the simplest top-level file with just what I needed.

I think we really need to avoid making low-level macros a kitchen sink that is used from a single-place and tons of options. That's not maintainable either.

It's a fine balance to reach indeed.

Reply:
@plbossart @ranj063 @kv2019i any update on alignment here ? Is this better than what we have today ?